tests/operators/test_convection.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+"""Tests for convection diagnostics."""
+
+import iris
+
+import CSET.operators.convection as convection
+
+
+def test_cape_ratio():
+    """Compare with precalculated ratio."""
+    precalculated = iris.load_cube("tests/test_data/ECFlagB.nc")
```

Comment:
It wouldn't be a bad idea. For now it is manageable, but as our tests grow it would be beneficial. I do wonder if at some point we want to move the test data out of the repo, to prevent it taking up most of the space.